modules/pkg/paramscreator.go

```diff
@@ -58,6 +73,10 @@ func (r *run) ParamsCreator() (interface{}, error) {
 		switch checkType {
 		case "name":
 			p.PkgMatch.Matches = append(p.PkgMatch.Matches, checkValue)
+		case "version":
+			p.VerMatch = checkValue
+		case "notver":
```

Comment:
What about supporting a `!` operator like the one in the content parameter of the file module?